stingray/events.py

```diff
@@ -0,0 +1,202 @@
+"""
+Definition of :class:`EventList`.
+
+:class:`EventList` is used to handle photon arrival times.
+"""
+from __future__ import (absolute_import, unicode_literals, division,
+                        print_function)
```

Comment:
If I remember correctly, we are now avoiding `unicode_literals`, because using the standard string for each python version overall more consistent.